content/applications/sales/sales/gelato.rst

```diff
@@ -4,4 +4,166 @@
 Gelato
 ======
 
-The documentation for Gelato will be added soon.
+Gelato is a global print-on-demand platform that integrates with Odoo to sync product catalogs and
+automate order fulfillment.
+
+Connecting Gelato's services with Odoo's **Sales** and **eCommerce** apps enables the following:
+
+- Sync Odoo sales orders with Gelato for automated order fulfillment
+- Create and manage Gelato products within Odoo; supports product variant and image sync
+- Configure delivery options in Odoo and receive order updates via webhooks.
+
+Configuration
+=============
+
+.. important::
+   The company information (Company Name and Billing address) in the Gelato account *must* match the
+   company information in the Odoo database in order for sales orders to be confirmed and sent to
+   Gelato for fulfillment.
+
+   .. image:: gelato/gelato-company.png
+      :alt: Company information in Gelato.
+
+   .. image:: gelato/odoo-company.png
+      :alt: Company information in Odoo.
+
+Configure API Keys and Webhooks in Gelato
+-----------------------------------------
+
+Before configuring the Gelato connector in Odoo, first obtain API credentials and webhooks from the
+Gelato account.
+
+API connectors enable Odoo **Sales** to send and receive data from Gelato for order processing,
+while webhooks provide real-time updates on order status and shipment tracking.
+
+API Key
+~~~~~~~
+
+An API Key is a unique authentication token that allows Odoo to securely communicate with Gelato’s
```

Comment:
Use a _normal_ quote instead of a smart quote `'`
```suggestion
An API Key is a unique authentication token that allows Odoo to securely communicate with Gelato's
```